client/layout/masterbar/masterbar-cart/masterbar-cart-button.tsx

```diff
@@ -66,6 +74,8 @@ export function MasterbarCartButton( {
 						cartKey={ selectedSiteId }
 						goToCheckout={ goToCheckout }
 						closeCart={ onClose }
+						onRemoveProduct={ onRemoveProduct }
```

Comment:
@sirbrillig, this change triggers the following TypeScript error:

```javascript
yarn tsc -p client/tsconfig.json --noEmit

client/layout/masterbar/masterbar-cart/masterbar-cart-button.tsx:77:7 - error TS2322: Type '{ selectedSiteSlug: string; cartKey: string | number; goToCheckout: (siteSlug: string) => void; closeCart: () => void; onRemoveProduct: ((uuid: string) => void) | undefined; onRemoveCoupon: (() => void) | undefined; }' is not assignable to type 'IntrinsicAttributes & { selectedSiteSlug: string; cartKey: string | number | undefined; goToCheckout: (siteSlug: string) => void; closeCart: () => void; }'.
  Property 'onRemoveProduct' does not exist on type 'IntrinsicAttributes & { selectedSiteSlug: string; cartKey: string | number | undefined; goToCheckout: (siteSlug: string) => void; closeCart: () => void; }'.

77       onRemoveProduct={ onRemoveProduct }
         ~~~~~~~~~~~~~~~
```

It can be seen on `trunk`, I'm not sure why the unit tests didn't catch it.

Reply:
🤔 I don't think that error actually exists. Sometimes `tsc` gets confused because some packages haven't been recompiled since their API changed. Try running `yarn tsc -p packages/mini-cart` and then run the above check again on `client` and I suspect the error will have disappeared. I've noticed this sort of thing happen more lately. It looks like running `yarn build-packages` might also take care of this sort of thing, but it would be nice if this was built-in to our workflows somehow.

Reply:
I did as you said, and those errors indeed went away 👍.